packages/contentful-migrations/ui/1100000000001-create-m-stage-migration.js

```diff
@@ -146,12 +155,23 @@ module.exports = async function (migration, context) {
       linkType: 'Entry',
     });
 
-  mStage.changeFieldControl('name', 'builtin', 'singleLine', {});
-  mStage.changeFieldControl('title', 'builtin', 'singleLine', {});
-  mStage.changeFieldControl('subtitle', 'builtin', 'singleLine', {});
+  mStage.changeFieldControl('internal_name', 'builtin', 'singleLine', {
+    helpText: 'e.g. "Home page > Stage"',
+  });
+
+  mStage.changeFieldControl('overline', 'builtin', 'singleLine', {});
+
+  mStage.changeFieldControl('headline', 'builtin', 'singleLine', {});
+
+  mStage.changeFieldControl('subline', 'builtin', 'singleLine', {});
+
   mStage.changeFieldControl('text', 'builtin', 'richTextEditor', {});
+
   mStage.changeFieldControl('media', 'builtin', 'entryLinkEditor', {});
+
   mStage.changeFieldControl('links', 'builtin', 'entryLinksEditor', {});
+
   mStage.changeFieldControl('layout', 'builtin', 'dropdown', {});
```

Comment:
Maybe disable layout for the stage module?

Reply:
Fits better in the following "template clean up" task, as we currently have two layouts for the stage (fullscreen and split). This would then also have to be adapted in the template/styles.

Reply:
See #32